mkosi/sandbox.py

```diff
@@ -524,11 +524,15 @@ def execute(self, oldroot: str, newroot: str) -> None:
         # /dev/null fails with EACCESS for unknown reasons.
         mount("tmpfs", dst, "tmpfs", 0, "mode=0755")
 
-        for node in ("null", "zero", "full", "random", "urandom", "tty"):
+        for node in ("null", "zero", "full", "random", "urandom", "tty", "fuse"):
+            nsrc = joinpath(oldroot, "dev", node)
+            if not os.path.exists(nsrc):
+                continue
+
             ndst = joinpath(dst, node)
             os.close(os.open(ndst, os.O_CREAT | os.O_CLOEXEC | os.O_EXCL))
 
-            mount(joinpath(oldroot, "dev", node), ndst, "", MS_BIND, "")
+            mount(nsrc, ndst, "", MS_BIND, "")
```

Comment:
This means we silently ignore all missing device nodes instead of just /dev/fuse. We should fail loudly if any of the other device nodes are not available, so please rework this so that only /dev/fuse is skipped if it is not available.